### DISCUSSION TITLE
[Feature] Delete source/original video after encoding

### DISCUSSION THREAD

**Discussion by ockentap** [immich-app/immich#5266] [Feature Request]
### The feature

First of all i want to say thank you for an amazing piece of software, Immich is just amazing! But one thing is missing imo.

It would be amazing if you could switch out the original video with the encoded one, similar to how you by default can compress videos during upload to Google Photos. Most family style videos dont need to be high resolution or bitrate, the videos are already encoded to VP9 so why not give the user the ability to just delete the source? Currently you need to run unmanic or similar software in parallel with Immich which is unnecessary since everything is there.

Just add a button for the system admin to choose to delete source files after successful encoding. 

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
I'm going to close this as a duplicate of #1202, we're not planning to do this for now.
